odata-data-aggregation-ext/7 Examples.md

```diff
@@ -46,7 +46,7 @@ results in
 }
 ```
 
-Since `groupby` expands navigation properties in grouping properties by default, this is the same result as if the request would include a `$expand=Customer($select=Name)`. With the use of `groupby` the `$select` is not needed, because `groupby` removes all other properties.
+Since `groupby` expands navigation properties in grouping properties by default, this is the same result as if the request would include a `$expand=Customer($select=Name)`. Here `$select` is not needed, because `groupby` removes all other properties.
```

Comment:
Not so sure, if the binding between `Here` in statement 2 and `Since` in statement 1 has the same clarity as the mildly repeating construct before.